addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -314,14 +323,40 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Wait reloading...",
             trigger:  "html:not(:has(#completlyloaded)) div",
+        },
+        // Check that if we edit again and save again the default value is not deleted.
+        {
+            content: 'Enter in edit mode again',
+            trigger: 'a[data-action="edit"]',
+            run: 'click',
+        },
+        {
+            content: 'Verify that the editor appeared',
+            trigger: 'button[data-action="save"]',
+            run: function () {}
+        },
+        {
+            content: 'Edit the form',
+            trigger: '.s_website_form_field:eq(0) input',
+            run: 'click',
+        },
+        ...addCustomField('many2one', 'select', 'Select Field', true),
+        {
+            content: 'Save the page',
+            trigger: 'button[data-action=save]',
+            run: 'click',
+        },
+        {
+            content: 'Verify that the value has not been deleted',
+            trigger: '.s_website_form_field:eq(0) input[value="John Smith"]',
```

Comment:
In the last commit I see this:
![image](https://user-images.githubusercontent.com/10338094/157033141-bf4dcbe0-2eb8-4bf4-86dd-ace06b0c720a.png)
Is this actually needed to wait for the reload this way (and if so, should not it be the same here?)